doc/api/api_changes.rst

```diff
@@ -15,13 +15,140 @@ For new features that were added to matplotlib, please see
 Changes in 2.0.0
 ================
 
+
+
+Deprecation and removal
+-----------------------
+
 Color of Axes
--------------
+`````````````
 
 The ``axisbg`` and ``axis_bgcolor`` properties on ``Axes`` have been
 deprecated in favor of ``facecolor``.
 
 
+GTK and GDK backends deprecated
+```````````````````````````````
+The untested and broken GDK and GTK backends have been deprecated.
+These backends allows figures to be rendered via the GDK api to
+files and GTK2 figures. They are untested, known to be broken and
+use have been discouraged for some time. The `GTKAgg` and `GTKCairo` backends
+provide better and more tested ways of rendering figures to GTK2 windows.
+
+WX backend deprecated
+`````````````````````
+The untested WX backend has been deprecated.
+This backend allows figures to be rendered via the WX api to
+files and Wx figures. It is untested, and
+use have been discouraged for some time. The `WXAgg` backend
+provides a better and more tested way of rendering figures to WX windows.
+
+CocoaAgg backend removed
+````````````````````````
+
+The deprecated and not fully functional CocoaAgg backend has been removed
+
+
+General
+-------
+
+`Artist.update` has return value
+````````````````````````````````
+
+The methods `matplotlib.artist.Artist.set`,
+`matplotlib.Artist.update`, and the function `matplotlib.artist.setp`
+now use a common codepath to look up how to update the given artist
+properties (either using the setter methods or an attribute/property).
+
+The behavior of `matplotlib.Artist.update` is slightly changed to
+return a list of the returned values from the setter methods to avoid
+changing the API of `matplotlib.Artist.set` and
+`matplotlib.artist.setp`.
+
+The keys passed into `matplotlib.Artist.update` are now converted to
+lower case before being processed to match the behavior of
+`matplotlib.Artist.set` and `matplotlib.artist.setp`.  This should not
+break any user code because there are no set methods with capitals in
+the names, however going forward this puts a constraint on naming
+properties.
+
+
+`Legend` initializers gain edgecolor and facecolor kwargs
+``````````````````````````````````````````````````````````
+
+The :class:`~matplotlib.legend.Legend` background patch (or 'frame')
+can have its `edgecolor` and `facecolor` determined by the
+corresponding keyword arguments to its initializer, or to any of the
```

Comment:
The first "its" refers to the patch, the second to the legend; you could change 'to its initializer' to 'to the Legend initializer'.